release.py

```diff
@@ -62,8 +62,10 @@ def create_data_schema_documentation():
 
 def make_release_specific_changes(release_version):
     execute_steps([
-        "python ./web_frontend/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
-        "python ./osmaxx_conversion_service/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
+        "python ./web_frontend/manage.py makemessages -l en",
+        "python ./osmaxx_conversion_service/manage.py makemessages -l en",
```

Comment:
Good catch, they should actually write to the same file, and since they have somewhat the same codebase, this `should` produce essentially the same file content. Unfortunately this is not the case, because each has different apps loaded.

-> I would argue, that this needs some cleanup: Only the translations of the customer facing website should even be considered.